docs/developer-docs/daos/sns/launching/launch-summary-1proposal.mdx

```diff
@@ -90,6 +90,12 @@ If successful, at the end of stage, the following has changed:
   </tr>
 </table>
 
+:::warning
+Adding NNS Root as co-controller is necessary because NNS Root needs to be able to make sure that no one else has the ability to change the canisters during the swap. This is automatically done in a later step which sets NNS Root and SNS Root as the only controllers of the dapp canisters. The prohibition of changing the canisters during the swap is necessary to ensure that the swap participants get what they expect.
```

Comment:
```suggestion
Adding NNS root as co-controller is necessary because it needs to be able to make sure that no one else has the ability to change the canisters during the swap to ensure that the swap participants get what they expect. This is automatically done in a later step which sets NNS root and SNS root as the only controllers of the dapp canisters.
```